python/cugraph/tests/utils.py

```diff
@@ -20,29 +20,15 @@
 from cugraph.dask.common.mg_utils import (get_client)
 
 #
-# Datasets are numbered based on the number of elements in the array
+# Datasets
 #
-DATASETS_1 = ['../datasets/netscience.csv']
+DATASETS_UNDIRECTED = ['../datasets/karate.csv',  '../datasets/dolphins.csv']
+DATASETS_UNRENUMBERED = ['../datasets/karate-disjoint.csv']
 
-DATASETS_2 = ['../datasets/karate.csv',
-              '../datasets/dolphins.csv']
-
-DATASETS_3 = ['../datasets/karate.csv',
-              '../datasets/dolphins.csv',
-              '../datasets/email-Eu-core.csv']
-
-# FIXME: netscience.csv causes NetworkX pagerank to throw an exception.
-# (networkx/algorithms/link_analysis/pagerank_alg.py:152: KeyError: 1532)
-DATASETS_4 = ['../datasets/karate.csv',
-              '../datasets/dolphins.csv',
-              '../datasets/netscience.csv',
-              '../datasets/email-Eu-core.csv']
-
-DATASETS_5 = ['../datasets/karate.csv',
-              '../datasets/dolphins.csv',
-              '../datasets/polbooks.csv',
-              '../datasets/netscience.csv',
-              '../datasets/email-Eu-core.csv']
+DATASETS = ['../datasets/karate-disjoint.csv',
+            '../datasets/dolphins.csv',
+            '../datasets/netscience.csv',
+            '../datasets/email-Eu-core.csv']
 
 STRONGDATASETS = ['../datasets/dolphins.csv',
```

Comment:
Why is dolphin a "strong" dataset? It looks a bit redundant with Karate to me. We could consider getting rid of it if we want to do some dataset cleanup.